DEPENDENCIES.md

```diff
@@ -34,7 +34,7 @@
 |--------------------|-------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | oc (OpenShift CLI) | 4.10              | To interact with the cluster                                                | Follows OpenShift version                                                                                                                                        |
 | kubectl            | 1.23.x            | To interact with the cluster                                                | Follows kubernetes version which follows OpenShift version. We only need either oc or kubectl                                                                    |
```

Comment:
kubectl needs to be moved to 1.24 as kcp 0.7 is based on 1.24. OpenShift 4.10 is based on 1.23 but it is best to have the version mismatch (n-2 compatibility is guaranteed IIRC) in this direction not to miss any new feature.

Reply:
see also: https://github.com/openshift-pipelines/pipeline-service/pull/193#discussion_r946322220